MAINTAINERS.md

```diff
@@ -1,17 +1,91 @@
+publication of the SDK and its individual packages.
+
+### Prerequisites
+
+#### NPM Account
+
+Sign up for an [NPM account](https://www.npmjs.com/signup) with
+some form of 2FA. Then request that an administrator add your NPM
+user ID to the `@agoric` organization and the `agoric` package.
+
+### Preparation
+
+Together with the Release Owner and other appropriate stakeholders,
+ensure there is consensus on the following parameters for the
+release:
+
+- Base branch: which branch will the release be based on? The base
+branch will need to be stable and may be frozen at various times,
+so it is a good practice to have the base branch be a branch off
+of `master`. The base branch might also be used for followon patch
+releases before the next major release.
+
+- Release labels: we currently label our releases with both a
+human-meaningful label such as `pismoA` and a sequential upgrade
+label like `agoric-upgrade-8`.  In the future, semantic versioning
+might replace either or both label styles.
+
+- Development history oriented release descrition: a short description
+of the major changes in the release from the point of view of
+understanding the source code and tracked issues addressed by the
+release.
+
+- Validator oriented release description: a short description of the
+release from the perspective of a validator to understand why the
+release should be installed and important operational changes.
+
+### Staging the Release
+
+Until we reach the next section "Publish the release", the release
+process can be aborted.
+
+- [ ] Check the development history oriented release description
+into the top-level CHANGELOG.md on the base branch.
+
+- [ ] Review the [next release](
 https://github.com/Agoric/agoric-sdk/labels/next%20release
 ) label for additional tasks particular to this release.
 
+- [ ] Create a fresh, clean clone of the repository.
+
+This avoids accidental contamination from untracked or ignored
+changes in one's usual development repository.
+
+```sh
+# Fresh checkout under ~/release or any other preferred directory without agoric-sdk
+cd ~/release
+git clone https://github.com/Agoric/agoric-sdk.git
+cd agoric-sdk
+```
+
+- [ ] Check out the appropriate base branch as outlined in "Preparation" above.
+
+- [ ] Create a release branch tagged with a timestamp. The specific commit tagged as the release will live
+in this branch. The release tags will be human-meaningful, the release branch need not be.
+
 ```sh
 # Create a release branch.
 now=`date -u +%Y%m%dT%H%M%S`
 git checkout -b release-$now
 ```
 
-To generate a new final release, and CHANGELOG.md files
-(use `--conventional-prerelease` instead of `--conventional-graduate` if you just want to generate a dev release):
+- [ ] Do a `yarn install` to generate tooling needed for the release.
+
+```sh
+# yarn install to build release tools
+yarn install --force
+```
+
+- [ ] Generate new SDK version and per-package CHANGELOG.md files
```

Comment:
it might be good to explain that this step (specifically the `yarn lerna version` will:
* modify the `package.json` (to bump the version) of every package which saw changes since the previous release, or whose dependencies change because of other packages being bumped
  * in practice this means pretty much every package in the monorepo
* update all dependencies in the monorepo to match the bumped versions
* create CHANGELOG.md files for each package with a summary of the git commit history
* make a `git commit` with those changes
* create a `$package@$version` git tag for every package that changed

The first time I walked through this process, I was surprised by how much the repo changed during the execution of the `lerna version` step.

Reply:
Done.